src/app/components/LayerStatus/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import CheckCircleIcon from '@mui/icons-material/CheckCircle'
+import ErrorIcon from '@mui/icons-material/Error'
+import Typography from '@mui/material/Typography'
+import { COLORS } from '../../../styles/theme/colors'
+
+type LayerStatusProps = {
+  valid: boolean
```

Comment:
I would prefer calling it `uptodate` (or `upToDate`) instead of `valid`, because invalid is too strong to describe a state that is not fully uptodate...

Reply:
Ahh, I see it has been renamed in the next commit. Never mind then.